system/Database/BaseBuilder.php

```diff
@@ -445,6 +445,24 @@ public function selectCount(string $select = '', string $alias = '')
         return $this->maxMinAvgSum($select, $alias, 'COUNT');
     }
 
+    /**
+     * Adds a subquery to the selection
+     *
+     * @return static
```

Comment:
```suggestion
     * @return $this
```

Reply:
If you specify a return type after the method, then the rector removes the @return from the docblock.
```
     /**
      * Adds a subquery to the selection
      *
-     * @return static
      */
     public function selectSubquery(BaseBuilder $subquery, string $as): self
     {
    ----------- end diff -----------

Applied rules:
 * RemoveUselessReturnTagRector

```

Reply:
Okay.